<?xml version="1.0"?>
<launch>
  <node pkg = "cloud_factory" type = "random_clouds" name = "random_clouds">
    <param name = "topic_cloud" type = "string" value = "transformed_cloud"/>
    <param name = "frame_cloud" type = "string" value = "map"/>
    <param name = "frame_rate" type = "double"  value = "1.0"/>
  </node>
</launch>